cloudify_agent/installer/__init__.py

```diff
@@ -341,16 +342,21 @@ def create_custom_env_file_on_target(self, environment):
             return None
 
     def download(self, url, destination=None):
+        local_cert_file = self.cloudify_agent.get('local_rest_cert_file')
+        verify_cert = self.cloudify_agent.get('verify_rest_certificate')
+        if verify_cert is None:
```

Comment:
and by the way according to daemons.py it is always set which is why `[]` is used instead of `get`